### PR TITLE
Switch from stale app to stale GHA

### PR DESCRIPTION
**Pull request by dfarrell07**
The stale app has been deprecated and replaced with a GitHub Action.

probot/probot.github.io#376
probot/stale#385

### CODE REVIEW

**Review by skitt — APPROVED** [review]
Note for other reviewers: this uses the default delay before closing stale issues/PRs, 7 days.

**Comment by dfarrell07** [comment]
> Note for other reviewers: this uses the default delay before closing stale issues/PRs, 7 days.

Right, which is the same we had manually configured before.

**Review by sridhargaddam — APPROVED** [review]

File: .github/workflows/stale.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      issues: write
+      pull-requests: write
+    steps:
+      - uses: actions/stale@b69b346013879cedbf50c69f572cd85439a41936
+        with:
+          days-before-issue-stale: 120
+          days-before-pr-stale: 14
+          exempt-issue-labels: 'confirmed,security'
+          exempt-pr-labels: 'confirmed,security'
+          stale-issue-label: 'stale'
```
Comment (tpantelis) [inline .github/workflows/stale.yml:25]: Should we use `wontfix` as before?

Comment (dfarrell07) [reply .github/workflows/stale.yml:25]: We can choose. Thought "stale" made more sense, was a more accurate description.

File: .github/workflows/stale.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+          stale-issue-message: |
+            This pull request has been automatically marked as stale because it has not had recent activity. It will be closed if no further
```
Comment (dfarrell07) [inline .github/workflows/stale.yml]: I should change this to "This issue..."

File: .github/workflows/stale.yml (hunk 1 above)
Comment (dfarrell07) [reply .github/workflows/stale.yml:25]: I'll create "stale" labels as I send these changes to repos. Already created one for this repo. I'll also update the scrub filter accordingly. We can clear the old "wolntfix" ones during the next scrub calls.

**Review by tpantelis — APPROVED** [review]

**Pull request merged**